addon/validators/sometimes.js

```diff
@@ -5,7 +5,9 @@ export default function validateSometimes(validators, condition) {
     return function(key, newValue, oldValue, changes, content) {
       let thisValue = {
         get(property) {
-          return get(changes, property) || get(content, property);
+          return get(changes, property) != null
```

Comment:
This will work for `false`, but not `undefined` or `null`. How about this instead?

```
let thisValue = {
  get(property) {
    if (property.includes('.')) {
      return get(changes, property) || get(content, property);
    }

    if (changes.hasOwnProperty(property)) {
      return get(changes, property);
    } else {
      return get(content, property);
    }
  }
};
```

If the property exists in changes, then return that. This way, it should work with `undefined` and `null` too. Otherwise, return the property from `content`. 

I'm not sure how best to handle property paths (i.e. `this.get('a.b')`), so that might need some more thinking, hence the check for a dot in the first conditional.

Reply:
This has just become an issue for us. Setting a property to `null` doesn't behave as you'd expect.